addons/mail/static/tests/tours/discuss_channel_public_tour.js

```diff
@@ -66,6 +66,47 @@ registry.category("web_tour.tours").add("discuss_channel_public_tour.js", {
             content: "Check message contains the attachment",
             trigger: '.o-mail-Message .o-mail-AttachmentCard:contains("text.txt")',
         },
+        {
+            content: "Click on add reaction",
```

Comment:
these comments can be removed, they make the test harder to read when the triggers are self explanatory and add more maintenance over time as they can become obsolete